javaslang-benchmark/src/test/java/javaslang/benchmark/JmhRunner.java

```diff
@@ -0,0 +1,73 @@
+package javaslang.benchmark;
+
+import org.openjdk.jmh.annotations.Mode;
+import org.openjdk.jmh.results.RunResult;
+import org.openjdk.jmh.runner.Runner;
+import org.openjdk.jmh.runner.options.Options;
+import org.openjdk.jmh.runner.options.OptionsBuilder;
+import org.openjdk.jmh.runner.options.TimeValue;
+
+import java.util.Collection;
+import java.util.concurrent.TimeUnit;
+
+public class JmhRunner {
+    private static final int WARMUP_ITERATIONS = 10;
+    private static final int MEASUREMENT_ITERATIONS = 40;
```

Comment:
We need it for proper statistical analysis imo. Even without GC, the swings are still considerable and therefore when running with 10 iterations the margin of error is still high.  Also remember each iteration is just 500ms.

I think it is ok to use `JmhRunner.devRun` during development for quick feedback (10 iterations). 

However, if we are gathering reports for publication or to want to make certain decisions, I think 40 is a good number.